### PR TITLE
issue/pr templates

### PR DESCRIPTION
**Pull request by juliusmarminge**
Related issue #2356 

Added template forms for Issues and template markdown for Pull Requests.

Test Issue forms here: https://github.com/juliusmarminge/test-github-stuff/issues/new/choose

Also added actions workflow to automatically label stuff

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+What changes are made in this PR? Is it a feature or a bug fix?
+
+## ✅ Checklist
+
+- [ ] I have followed the steps listed in the [Contributing guide](/CONTRIBUTING.md).
+- [ ] If necessary, I have added documentation related to the changes made.
+
+## 🪪 Related Issues and Discussions
+
+Please link issues related to this pull request here.
```
Comment (sachinraja) [inline .github/pull_request_template.md:12]: It'll likely only be one issue so we may just want to have a `closes #` at the top of the template. Like this:
```
closes #

## 🎯 Changes
...
```

**Review by sachinraja — APPROVED** [review]
Great job, this will help a lot!

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      label: Describe the bug
+      description: Please describe the bug as clear and concise as possible.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: Please describe the behavior you were expecting when encountering the bug.
+    validations:
+      required: true
```
[the remark this answers is not in the document]
Comment (juliusmarminge) [reply .github/ISSUE_TEMPLATE/1.bug_report.yml]: > Great job, this will help a lot!

Made the changes you suggested 👍🏼

**Comment by juliusmarminge** [comment]
> Great job, this will help a lot!

Made the suggested changes 👍🏼

**Review by sachinraja — APPROVED** [review]

**Review by KATT — APPROVED** [review]
this is aaawesoooome!

**Pull request merged**